Source/WebGPU/WebGPU/Pipeline.mm

```diff
@@ -42,7 +42,8 @@
     }
 
     auto* ast = shaderModule.ast();
-    RELEASE_ASSERT(ast);
+    if (!ast)
+        return std::nullopt;
```

Comment:
I guess I was going by a different approach, and making sure that `createLibrary` is never called with an invalid module, but I think either option is fine. FWIW we do return nullopt below, when we can't find the entry point.

Reply:
Oh I'll do that instead. Indeed we shouldn't call this with an invalid shader module, maybe that is the root cause here